### PR TITLE
aix71 PR

### PR DESCRIPTION
**Pull request by Lex-2008**

### CODE REVIEW

**Comment by cf-bottom** [comment]
Thank you for submitting a pull request! Maybe @craigcomstock can review this?

**Comment by Lex-2008** [comment]
testbuild: https://ci.cfengine.com/job/pr-pipeline/2352/

File: deps-packaging/pkg-get-src [hunk 1]
```diff
@@ -120,7 +120,7 @@ get_src()
     else
         [ -f "$local_filename" ]  &&  rm -f "$local_filename"
 
-        tmp_filename=`fetch_file $FILENAME $CHECKSUM`
+        tmp_filename=`fetch_file "$FILENAME" $CHECKSUM`
```
Comment (Lex-2008) [inline deps-packaging/pkg-get-src]: "quote arg" com msg is too shrt

File: deps-packaging/libgcc/cfbuild-libgcc.spec [hunk 2]
```diff
@@ -23,7 +23,7 @@ rm -rf $RPM_BUILD_ROOT
 
 mkdir -p $RPM_BUILD_ROOT%{prefix}/lib
 
-cp  /opt/freeware/lib/gcc/powerpc-ibm-aix*.0.0/4.*/libgcc_s.a $RPM_BUILD_ROOT%{prefix}/lib/
+cp  /opt/freeware/lib/gcc/powerpc-ibm-aix*.0.0/[64].*/libgcc_s.a $RPM_BUILD_ROOT%{prefix}/lib/
```
Comment (Lex-2008) [inline deps-packaging/libgcc/cfbuild-libgcc.spec]: "Use AIX 7.1 libgcc if available" commit can be squashed with "other fixes for aix" commit which edits same line and has much worse commit message

**Comment by vpodzime** [comment]
> I'd approve this, but github doesn't allow me to, since I'm the creator of this PR

I can approve it myself then! :)

**Pull request merged**